File: bus-gradle-plugin/src/main/java/com/blankj/utilcode/util/BusUtils.java
package com.blankj.utilcode.util;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

public final class BusUtils {

    @Target({ElementType.METHOD})
    @Retention(RetentionPolicy.CLASS)
    public @interface Subscribe {
        String name() default "";
    }
}

File: utilcode/src/main/java/com/blankj/utilcode/util/BusUtils.kt
//package com.blankj.utilcode.util
//
//import java.lang.annotation.ElementType
//import java.lang.annotation.RetentionPolicy
//
//
//class BusUtils private constructor() {
//
//    init {
//        throw UnsupportedOperationException("u can't instantiate me...")
//    }
//
//    companion object {
//        fun <T> post(name: String?, vararg objects: Any): T {
//            return if (name == null || name.length == 0) null else null
//        }
//
//    }
//
//    @Target(ElementType.METHOD)
//    @Retention(RetentionPolicy.CLASS)
//    annotation class Subscribe(val name: String = "")
//}